src/test/java/com/minecolonies/coremod/entity/ai/util/ChatSpamFilterTest.java

```diff
@@ -29,79 +30,79 @@ public void setup()
     @Test
     public void testFilterFirst()
     {
-        filter.requestWithoutSpam(MESSAGE_1);
+        filter.requestWithoutSpam(new TextComponentString(MESSAGE_1));
```

Comment:
I just took a look at the code.
Not possible, because we can't differentiate later if we need to have a String or a translation.